src/WebJobs.Script.WebHost/Metrics/FlexConsumptionMetricsPublisher.cs

```diff
@@ -305,6 +305,13 @@ public void AddFunctionExecutionActivity(string functionName, string invocationI
             // nothing to do here - we only care about Started/Completed events.
         }
 
+        // Rounds the given metric to the specified granularity in the options. This function should only be used for metrics expressed in ms.
+        private long RoundMS(long metricMS, int granularity)
+        {
+            var excess = metricMS % granularity;
```

Comment:
I found this calculation involving excess and the ternary operator to be slightly unintuitive.

Do you think this version is any clearer?

`return Math.Ceiling(metricMS / granularity) * granularity;` 

Also maybe update the method name to be clearer that the intent is to always round up?

Reply:
I think we'll have to cast `metricMS` to be a double before applying `Math.Ceiling`, but I think your version is clearer in intent. I'll update and adjust the method name as well